libs/@blockprotocol/graph/package.json

```diff
@@ -23,49 +23,75 @@
   },
   "type": "module",
   "exports": {
-    ".": "./dist/index.js",
-    "./codegen": [
-      "./dist/codegen.js"
-    ],
-    "./custom-element": "./dist/custom-element.js",
-    "./graph-service-json": "./dist/graph-service-json.js",
-    "./internal": "./dist/internal.js",
-    "./react": "./dist/react.js",
-    "./stdlib": [
-      "./dist/stdlib.js"
-    ]
+    ".": {
+      "types": "./dist/cjs/index.d.ts",
+      "require": "./dist/cjs/index.cjs",
+      "import": "./dist/es/index.js"
```

Comment:
Small thing, but I initially had this as `dist/esm/...` to be consistent with `core`, but `rollup` uses `es` as a format definition and I didn't want to complicate `rollup.config.js` needlessly with a ternary to change that in the filepath